backend/src/api/helper.clj

```diff
@@ -18,15 +18,19 @@
 (defn get-game-as-player
```

Comment:
Oh wow, this function hurts my eyes, I forgot about it (mostly the part I wrote hurts).

Can we extract all the keys that are more than single lines (so :rows, :winner and :scores) into private functions? I won't be able to sleep otherwise.

Reply:
Changed for your well-being